src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,16 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
The logger is incorrectly referencing the `AuthenticationServiceImpl.class` instead of the `OrderServiceImpl.class`. You should reference the current class to ensure that log messages are correctly associated with the class they come from.